tests/examples-smoke-tests/utils.ts

```diff
@@ -0,0 +1,126 @@
+import path from 'path';
+import execa from 'execa';
+import { promisify } from 'util';
+import _treeKill from 'tree-kill';
+import * as playwright from 'playwright';
+
+async function deleteAllData(projectDir: string) {
+  const { PrismaClient } = require(path.join(projectDir, './.keystone/prisma/generated-client'));
+
+  let prisma = new PrismaClient();
+
+  await Promise.all(Object.values(prisma).map((x: any) => x?.deleteMany?.()));
+
+  await prisma.$disconnect();
+}
+
+const treeKill = promisify(_treeKill);
+
+// this'll take a while
+jest.setTimeout(10000000);
+
+const promiseSignal = (): Promise<void> & { resolve: () => void } => {
+  let resolve;
+  let promise = new Promise<void>(_resolve => {
+    resolve = _resolve;
+  });
+  return Object.assign(promise, { resolve: resolve as any });
+};
+
+export const initUserTest = (getPage: () => playwright.Page) => {
```

Comment:
Help make it clear that this is testing the `initFirstItem` part of the API.

```suggestion
export const initFirstItemTest = (getPage: () => playwright.Page) => {
```